app/services/transaction_search/cross_core_searcher.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module TransactionSearch
+
+  class CrossCoreSearcher < BaseSearcher
+    def options
+      ["both", "yes", "no"]
+    end
+
+    def search(params)
+      return order_details if params.blank?
+
+      params = params.first
+
+      # Based on params, filter by cross_core_project_id present in order_detail.order
+      if params == "no"
+        order_details.joins(:order).where(orders: { cross_core_project_id: nil })
```

Comment:
I'm not 100% sure the desired logic for the queries here - can you ask the consortium about it?